### PR TITLE
Text file get!

### PR DESCRIPTION
**Pull request by MasaruMuda**

### CODE REVIEW

File: Marc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Wowee!
```
Comment (DavLac) [inline Marc:1]: Wowee is bad, you should say Wowaaa

Comment (MasaruMuda) [reply Marc:1]: omg you're right, will fix that!

File: Marc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Wowaaa!
```
Comment (NielUp) [inline Marc:1]: it should be wowawiewa (borat) (if you know you know)

**Review by NielUp — CHANGES_REQUESTED** [review]
it should be wowawiewa (borat) (if you know you know)

File: Marc (hunk 2 above)
Comment (MasaruMuda) [reply Marc]: Did it!

**Pull request merged**